lisa/operating_system.py

```diff
@@ -1404,10 +1398,9 @@ def _initialize_package_installation(self) -> None:
             #  issue
             cmd_results = self._node.execute("yum repolist -v", sudo=True)
             if 0 != cmd_results.exit_code:
-                cmd_results = self._node.execute(
-                    "yum-config-manager --save --setopt=skip_if_unavailable=true",
-                    sudo=True,
-                )
+                from lisa.tools import YumConfigManager
```

Comment:
Avoid importing inline. Move the tool to base_tools, which is used in operating_system. And then move the import line to the top of the file.

Reply:
update